src/core/logical_layers/renderers/BivariateRenderer.tsx

```diff
@@ -1,6 +1,8 @@
 import throttle from 'lodash/throttle';
 import isNil from 'lodash/isNil';
 import { h3ToGeoBoundary, geoToH3 } from 'h3-js';
+import { Popup } from 'maplibre-gl';
```

Comment:
How about
```suggestion
import { Popup as MapPopup } from 'maplibre-gl';
```
or smth
I didn't even knew it has this thing :D

Reply:
Nice! I would be more clear that it's not a usual popup. 👌